protos/sources/vega/vega.proto

```diff
@@ -155,6 +155,8 @@ enum AuctionTrigger {
   AUCTION_TRIGGER_LIQUIDITY_TARGET_NOT_MET = 5;
   // Liquidity auction due to not being able to deploy LP orders because there's nothing to peg on one or both sides of the book
   AUCTION_TRIGGER_UNABLE_TO_DEPLOY_LP_ORDERS = 6;
+  // Market is suspended and put into auction via governance
+  AUCTION_TRIGGER_GOVERNANCE_SUSPENSION = 7;
```

Comment:
I think we're going to need to add these new types to some datanode migration somewhere given they are defined as enums here:
https://github.com/vegaprotocol/vega/blob/develop/datanode/sqlstore/migrations/0001_initial.sql#L504-L506

Reply:
ugh